go/vt/dbconnpool/connection.go

```diff
@@ -36,24 +37,21 @@ func (dbc *DBConnection) handleError(err error) {
 
 // ExecuteFetch is part of PoolConnection interface.
 func (dbc *DBConnection) ExecuteFetch(query string, maxrows int, wantfields bool) (*proto.QueryResult, error) {
-	start := time.Now()
+	defer dbc.mysqlStats.Record("Exec", time.Now())
```

Comment:
Does that work? I thought the eval of time.Now() would be done at end of function time, not right now...